<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1491</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QGridLayout" name="centralLeft">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="plotAndVector">
        <item>
         <layout class="QHBoxLayout" name="plotLayout"/>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="minimumSize">
           <size>
            <width>375</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>498</width>
             <height>602</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="2">
             <widget class="QLabel" name="v_windSpeedAverage">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="l_name">
              <property name="text">
               <string>Nom de la donnée mesurée</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLabel" name="v_airPressure">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QCheckBox" name="c_hailIntensity">
              <property name="text">
               <string>Intensité grêle</string>
              </property>
             </widget>
            </item>
            <item row="7" column="2">
             <widget class="QLabel" name="v_rainAccumulation">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="u_relativeHumidity">
              <property name="text">
               <string>%RH</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="c_windDirectionAvrg">
              <property name="text">
               <string>Direction moyenne du vent</string>
              </property>
             </widget>
            </item>
            <item row="23" column="2">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="c_relativeHumidity">
              <property name="text">
               <string>Humidité relative</string>
              </property>
             </widget>
            </item>
            <item row="16" column="3">
             <widget class="QLabel" name="u_hailIntensity">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="20" column="0">
             <widget class="QCheckBox" name="c_supplyVoltage">
              <property name="text">
               <string>Tension d'alimentation</string>
              </property>
             </widget>
            </item>
            <item row="21" column="0">
             <widget class="QCheckBox" name="c_refVoltage">
              <property name="text">
               <string>Tension de référence</string>
              </property>
             </widget>
            </item>
            <item row="16" column="2">
             <widget class="QLabel" name="v_hailIntensity">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="17" column="0">
             <widget class="QCheckBox" name="c_heatTemperature">
              <property name="text">
               <string>Température de chauffage</string>
              </property>
             </widget>
            </item>
            <item row="8" column="2">
             <widget class="QLabel" name="v_rainDuration">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLabel" name="u_windSpeedAverage">
              <property name="text">
               <string>m/s..</string>
              </property>
             </widget>
            </item>
            <item row="12" column="2">
             <widget class="QLabel" name="v_rainIntensity">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QCheckBox" name="c_rainDuration">
              <property name="text">
               <string>Durée de la pluie</string>
              </property>
             </widget>
            </item>
            <item row="19" column="2">
             <widget class="QLabel" name="v_heatTemperature_2">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="14" column="2">
             <widget class="QLabel" name="v_hailAccumulation">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="14" column="3">
             <widget class="QLabel" name="u_hailAccumulation">
              <property name="text">
               <string>hits/cm2, ...</string>
              </property>
             </widget>
            </item>
            <item row="20" column="3">
             <widget class="QLabel" name="u_supplyVoltage">
              <property name="text">
               <string>V</string>
              </property>
             </widget>
            </item>
            <item row="20" column="2">
             <widget class="QLabel" name="v_supplyVoltage">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="19" column="3">
             <widget class="QLabel" name="u_heatTemperature_2">
              <property name="text">
               <string>V</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="u_airTemperture">
              <property name="text">
               <string>C, F</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QCheckBox" name="c_hailDuration">
              <property name="text">
               <string>Durée de la grêle</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="c_airTemperture">
              <property name="text">
               <string>Temperature de l'air</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="l_value">
              <property name="text">
               <string>Valeur</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="17" column="2">
             <widget class="QLabel" name="v_heatTemperature">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="c_windSpeedAverage">
              <property name="text">
               <string>Vitesse moyenne du vent</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="u_windDirectionAvrg">
              <property name="text">
               <string>deg</string>
              </property>
             </widget>
            </item>
            <item row="15" column="2">
             <widget class="QLabel" name="v_hailDuration">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="c_rainAccumulation">
              <property name="text">
               <string>Accumulation de pluie</string>
              </property>
             </widget>
            </item>
            <item row="12" column="3">
             <widget class="QLabel" name="u_rainIntensity">
              <property name="text">
               <string>mm/h</string>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QLabel" name="u_airPressure">
              <property name="text">
               <string>Pa, hPa, bar, ...</string>
              </property>
             </widget>
            </item>
            <item row="17" column="3">
             <widget class="QLabel" name="u_heatTemperature">
              <property name="text">
               <string>C, F</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QCheckBox" name="c_hailAccumulation">
              <property name="text">
               <string>Accumulation de grêle</string>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QLabel" name="u_rainAccumulation">
              <property name="text">
               <string>mm, in</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="l_unit">
              <property name="text">
               <string>Unité</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="v_windDirectionAvrg">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QCheckBox" name="c_rainIntensity">
              <property name="text">
               <string>Intensité pluie</string>
              </property>
             </widget>
            </item>
            <item row="19" column="0">
             <widget class="QCheckBox" name="c_heatVoltage">
              <property name="text">
               <string>Tension de chauffage</string>
              </property>
             </widget>
            </item>
            <item row="21" column="2">
             <widget class="QLabel" name="v_refVoltage">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="c_airPressure">
              <property name="text">
               <string>Pression de l'air</string>
              </property>
             </widget>
            </item>
            <item row="22" column="3">
             <widget class="QLabel" name="u_analog1">
              <property name="text">
               <string>Label</string>
              </property>
             </widget>
            </item>
            <item row="21" column="3">
             <widget class="QLabel" name="u_refVoltage">
              <property name="text">
               <string>V</string>
              </property>
             </widget>
            </item>
            <item row="8" column="3">
             <widget class="QLabel" name="u_rainDuration">
              <property name="text">
               <string>sec</string>
              </property>
             </widget>
            </item>
            <item row="22" column="0">
             <widget class="QCheckBox" name="c_analog1">
              <property name="text">
               <string>Entrée analogique</string>
              </property>
             </widget>
            </item>
            <item row="22" column="2">
             <widget class="QLabel" name="v_analog1">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="15" column="3">
             <widget class="QLabel" name="u_hailDuration">
              <property name="text">
               <string>sec</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLabel" name="v_relativeHumidity">
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="v_airTemperture">
              <property name="text">
               <string>Label</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>l_name</zorder>
           <zorder>l_value</zorder>
           <zorder>l_unit</zorder>
           <zorder>c_windDirectionAvrg</zorder>
           <zorder>v_windDirectionAvrg</zorder>
           <zorder>u_windDirectionAvrg</zorder>
           <zorder>c_windSpeedAverage</zorder>
           <zorder>v_windSpeedAverage</zorder>
           <zorder>u_windSpeedAverage</zorder>
           <zorder>v_airTemperture</zorder>
           <zorder>c_airTemperture</zorder>
           <zorder>u_airTemperture</zorder>
           <zorder>u_relativeHumidity</zorder>
           <zorder>c_relativeHumidity</zorder>
           <zorder>v_relativeHumidity</zorder>
           <zorder>c_airPressure</zorder>
           <zorder>v_airPressure</zorder>
           <zorder>u_airPressure</zorder>
           <zorder>u_rainAccumulation</zorder>
           <zorder>c_rainAccumulation</zorder>
           <zorder>v_rainAccumulation</zorder>
           <zorder>v_rainIntensity</zorder>
           <zorder>v_rainDuration</zorder>
           <zorder>c_rainDuration</zorder>
           <zorder>u_rainDuration</zorder>
           <zorder>u_rainIntensity</zorder>
           <zorder>u_hailAccumulation</zorder>
           <zorder>c_hailDuration</zorder>
           <zorder>v_hailAccumulation</zorder>
           <zorder>c_hailAccumulation</zorder>
           <zorder>v_hailDuration</zorder>
           <zorder>u_hailDuration</zorder>
           <zorder>v_hailIntensity</zorder>
           <zorder>u_hailIntensity</zorder>
           <zorder>u_heatTemperature</zorder>
           <zorder>v_heatTemperature</zorder>
           <zorder>c_hailIntensity</zorder>
           <zorder>c_heatTemperature</zorder>
           <zorder>u_supplyVoltage</zorder>
           <zorder>v_heatTemperature_2</zorder>
           <zorder>c_heatVoltage</zorder>
           <zorder>u_heatTemperature_2</zorder>
           <zorder>v_supplyVoltage</zorder>
           <zorder>c_supplyVoltage</zorder>
           <zorder>c_refVoltage</zorder>
           <zorder>u_refVoltage</zorder>
           <zorder>v_refVoltage</zorder>
           <zorder>v_analog1</zorder>
           <zorder>u_analog1</zorder>
           <zorder>c_analog1</zorder>
           <zorder>c_rainIntensity</zorder>
           <zorder>verticalSpacer_2</zorder>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>Commande spéciale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Envoyer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Réponse :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="centralRight">
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="l_delai">
           <property name="text">
            <string>Délai entre les acquisitons</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QDial" name="dial_delai_acquisition"/>
         </item>
         <item row="6" column="0">
          <widget class="QwtCompass" name="Compass">
           <property name="lineWidth">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="b_play">
             <property name="text">
              <string>►</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="b_pause">
             <property name="text">
              <string>║</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="b_arret">
             <property name="text">
              <string>■</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QDoubleSpinBox" name="dial_delai_acquisitionSpin"/>
           </item>
           <item>
            <widget class="QLabel" name="dial_delai_unity">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>sec</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1491</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguration_port_s_rie">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="m_portSerie"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
    <addaction name="m_about"/>
    <addaction name="m_aboutQt"/>
   </widget>
   <widget class="QMenu" name="menuCourbe">
    <property name="title">
     <string>Courbe</string>
    </property>
    <addaction name="m_clearCurve"/>
   </widget>
   <widget class="QMenu" name="menuDonnees">
    <property name="title">
     <string>Données</string>
    </property>
    <addaction name="m_saveData"/>
    <addaction name="m_loadData"/>
   </widget>
   <addaction name="menuConfiguration_port_s_rie"/>
   <addaction name="menuCourbe"/>
   <addaction name="menuDonnees"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="m_portSerie">
   <property name="text">
    <string>Port série</string>
   </property>
  </action>
  <action name="m_about">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="m_aboutQt">
   <property name="text">
    <string>A propos de  Qt</string>
   </property>
  </action>
  <action name="m_clearCurve">
   <property name="text">
    <string>Effacer</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="m_saveData">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="m_loadData">
   <property name="text">
    <string>Charger</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtCompass</class>
   <extends>QwtDial</extends>
   <header>qwt_compass.h</header>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
